code.md

```diff
@@ -0,0 +1,128 @@
+<div id ='placeHolder'>
+ <article id = article>
+  <h3 id ='title'>
+  <p id = 'summary'>
+
+   /*
+      let input = document.getElementById('userInput');
+       let filterTitle = input.value.toUpperCase(); 
+      let theFilmTitle = document.getElementsByTagName('h3');
+       for(let i = 0; i<theFilmTitle.length ;i++){
+           let textValue = theFilmTitle[i].textContent|| theFilmTitle[i].innerText;
+           if(textValue.toUpperCase().indexOf(filterTitle)>-1){
+             theFilmTitle[i].style.display = 'block';
+              }else{
+             theFilmTitle[i].style.display = 'none';
+              }
+            }
+      
+      let filterSummary = input.value.toUpperCase();
+      let theSummary = document.getElementsByTagName('p');
+      for (let j = 0; j < theFilmTitle.length; j++) {
+        let textValue = theSummary[j].textContent || theSummary[j].innerText;
+        if (textValue.toUpperCase().indexOf(filterSummary) > -1) {
+          theSummary[j].style.display = 'block';
+        } else {
+          theSummary[j].style.display = 'none';
+        }
```

Comment:
Here you are doing two loops with much repetition. Could you think of a way to combine these two into a single loop? (hint: do both of these loops have a parent element that we could use instead?)